azure_usage/src_webapp/totals.py

```diff
@@ -73,12 +86,46 @@ def group_year_month(raw_data):
     """
 
     raw_data_ = raw_data.copy()
-    raw_data_[CONST_COL_NAME_YM] = pd.to_datetime(raw_data_[CONST_COL_NAME_DATE]).apply(lambda x: '{year}-{month:02d}'.format(year=x.year, month=x.month))
+    raw_data_[CONST_COL_NAME_YM] = pd.to_datetime(raw_data_[CONST_COL_NAME_DATE]).apply(
+        lambda x: "{year}-{month:02d}".format(year=x.year, month=x.month)
+    )
 
-    raw_data_gr = raw_data_.groupby([CONST_COL_NAME_YM])[CONST_COL_NAME_COST].sum().reset_index().sort_values(by=[CONST_COL_NAME_YM], ascending=True)
+    raw_data_gr = (
+        raw_data_.groupby([CONST_COL_NAME_YM])[CONST_COL_NAME_COST]
+        .sum()
+        .reset_index()
+        .sort_values(by=[CONST_COL_NAME_YM], ascending=True)
+    )
 
     return raw_data_gr
 
+
+def group_year_month_service(raw_data):
```

Comment:
new function

Reply:
Could you please write unittests for the new function.